wg-batch/annual-report-2024.md

```diff
@@ -2,32 +2,136 @@
 
 ## Current initiatives and Project Health
 
-
 1. What work did the WG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - artifacts
-   - reports
-   - white papers
-   - work not tracked in KEPs
--->
+See [2024 Highlights](#2024-highlights).
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
+  Yes, JobSet has 1 active owner at the moment.
+
+### 2024 Highlights
+
+We will breakdown our highlights into Sub Projects, KEPs, talks, community adoption.
+
+#### Sub Projects
+
+##### Kueue
+
+Kueue has had 5 releases in 2024.
```

Comment:
```suggestion
Kueue has had 5 minor releases in 2024.
```

Actually, we have releases more than 5. But I agree with mentioning only minor versions. So, could we clarify that here?